### ISSUE TITLE
После обновы

### ISSUE THREAD

**Issue by darhanger** [shamhi/HamsterKombatBot#1545]
![image](https://github.com/user-attachments/assets/99cda54c-db3b-4299-a395-b58f054982f3)

**Comment by Hoochizoid** [comment]
Аналогичная ситуация.

**Comment by epmakOFF** [comment]
+

**Comment by Dumpff** [comment]
+

**Comment by waslost0** [comment]
![image](https://github.com/user-attachments/assets/151740a5-e37b-421f-b713-30270a246840)

**Comment by epmakOFF** [comment]
Теперь после [2da12fc](https://github.com/shamhi/HamsterKombatBot/commit/2da12fc6265b8870e94da4477acc5b422bd70e67):
```
2024-07-14 13:41:11 | ERROR    | 32 - Unknown error while getting Nuxt Builds: Cannot connect to host hamsterkombat.io:443 ssl:default [Name or service not known] | Response text: ...
2024-07-14 13:41:15 | ERROR    | 32 - Unknown error while getting Access Token: Cannot connect to host api.hamsterkombat.io:443 ssl:default [Name or service not known] | Response text: ...
2024-07-14 13:41:18 | ERROR    | 32 - Unknown error while getting Nuxt Builds: Cannot connect to host hamsterkombat.io:443 ssl:default [Name or service not known] | Response text: ...
2024-07-14 13:41:21 | ERROR    | 32 - Unknown error while getting Access Token: Cannot connect to host api.hamsterkombat.io:443 ssl:default [Name or service not known] | Response text: ...
```

**Comment by epmakOFF** [comment]
Also: 
Cannot connect to host hamsterkombat.io:443 #1546

**Comment by epmakOFF** [comment]
Fixed in 0f54aad8a7478b996eee15503635963931706245
